web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -378,6 +378,7 @@ input:not([disabled]):focus.usa-user-error {
 
   .usa-border-error {
     border-left: none;
+    padding-left: 1rem;
```

Comment:
I'm reviewing this and it works if you leave all the fields empty. If I fill in just one field and see this:

![image](https://github.com/user-attachments/assets/ba7cceba-3c72-4560-ad50-7303e3e12ef7)

It looks like in that case `div#error-border` isn't getting this error class applied.

Reply:
might be more work to make them consistent, but check out how errors are marked on`/site-issue-report-form`. The error line has the correct padding but does not look like a single connected line